pmultiqc/modules/quantms/quantms.py

```
@@ -119,63 +112,136 @@ def __init__(
+        self.pep_quant_table = dict()
+        self.mzml_table = OrderedDict()
+        self.search_engine = OrderedDict()
+        self.xcorr_hist_range = {"start": 0, "end": 5, "step": 0.1}
+        self.hyper_hist_range = {"start": 0, "end": 5, "step": 0.1}
+        self.spec_evalue_hist_range = {"start": 0, "end": 20, "step": 0.4}
+        self.pep_hist_range = {
+            "start": 0,
+            "end": 1,
+            "low_thresh": 0.3,
+            "high_thresh": 0.7,
+            "end": 1,
```

Ruff (0.13.1)

Dictionary key literal `"end"` repeated

Remove repeated key literal `"end"`

(F601)